CHANGELOG.md

```diff
@@ -2,12 +2,20 @@
 
 ## Unreleased
 
+## 🛑 Breaking changes 🛑
+
+### 🚀 New components 🚀
+
+### 💡 Enhancements 💡
+
+### 🧰 Bug fixes 🧰
+
+## v0.52.0
+
 ### 🛑 Breaking changes 🛑
 
 - `jmxreceiver`: Remove properties & groovyscript parameters from JMX Receiver. Add ResourceAttributes & LogLevel parameter to supply some of the removed functionality with reduced attack surface (#9685)
 
-### 🚩 Deprecations 🚩
```

Comment:
Should we keep Deprecations in Unreleased?

Reply:
Done 👍